app/build.gradle

```diff
@@ -99,13 +98,17 @@ android {
             versionNameSuffix '-DEBUG'
         }
     }
+    buildFeatures {
+        viewBinding true
```

Comment:
Did a little bit of read and saw that viewBinding is much more performant compared to KAE as well. Nice!